.pipelines/singletenancy/cilium-overlay/cilium-overlay-e2e-step-template.yaml

```
@@ -140,7 +140,32 @@ steps:
       bash hack/scripts/validate_state.sh
     name: "validatePods"
     displayName: "Validate Pods"
-  
+
+  - script: |
+      echo "Run wireserver and metadata connectivity Tests"
+      cat test/integration/manifests/jobs/wget-job.yaml | sed 's/{address}/168.63.129.16/g; s/{path}/machine\/plugins?comp=nmagent\&type=getinterfaceinfov1/g'|kubectl apply -f -
```

I understand that how we are using the wget. But can you please explain `s/{address}/168.63.129.16/g; s/{path}/machine\/plugins?comp=nmagent\&type=getinterfaceinfov1/g`  what is the usecase address we are trying to hit and what is the response ?

---

when the sed strings have `/` in them, you can use a different separator:
```suggestion
      cat test/integration/manifests/jobs/wget-job.yaml | sed 's,{address},168.63.129.16,g; s,{path},machine/plugins?comp=nmagent\&type=getinterfaceinfov1,g'|kubectl apply -f -
```
might be easier to read